core/output/slack.js

```diff
@@ -1,28 +1,29 @@
 var express = require('express'),
 	bodyParser  = require("body-parser"),
 	request = require('request'),
+	shim = require("../shim.js"),
 	app = null,
-   	server,
-	sendMessage = function(text, thread) {
+  server = null,
+	platform = null,
+
+	sendMessage = function(message, thread) {
         var slackTeams = exports.config.slack_teams,
-		    slack_token = null,
-            thread_components = thread.split('~', 2),
-            thread_id = thread_components[0],
-            thread_team_id = thread_components[1];
-
-        for (var i = 0; i < slackTeams.length; i++) {
-            if (slackTeams[i].slack_team_id == thread_team_id) {
-                slack_token = slackTeams[i].slack_token;
-                break;
-            }
-        }
+		    	slack_token = null,
+          thread_components = thread.split('~', 2),
+          thread_id = thread_components[0],
+          thread_team_id = thread_components[1];
+
+				slack_token = slackTeams[thread_team_id].slack_token;
```

Comment:
why assign to null then immediately replace its value with something else? Just assign directly